rpm_base_4repos_rerun_changes/generate.sh

```diff
@@ -0,0 +1,36 @@
+sudo systemctl stop httpd pulp_workers pulp_resource_manager pulp_celerybeat
+mongo pulp_database --eval "db.dropDatabase()"
+sudo rm -rf /var/lib/pulp/content
+sudo rm -rf /var/lib/pulp/published
+sudo -u apache pulp-manage-db;
+sudo systemctl start httpd pulp_workers pulp_resource_manager pulp_celerybeat
+pulp-admin login -u admin -p admin
+pulp-admin rpm repo create --repo-id rpm-empty
+pulp-admin rpm repo create --feed https://fixtures.pulpproject.org/rpm-with-modules/ --repo-id rpm-with-modules --download-policy on_demand
+pulp-admin rpm repo create --feed https://fixtures.pulpproject.org/rpm-distribution-tree/ --repo-id rpm-distribution-tree --download-policy on_demand
+pulp-admin rpm repo create --feed https://fixtures.pulpproject.org/srpm-unsigned/ --repo-id srpm-unsigned --download-policy on_demand 
+# need to wait for tasks to finish
+pulp-admin rpm repo sync run --repo-id rpm-with-modules &
+pulp-admin rpm repo sync run --repo-id rpm-distribution-tree &
+pulp-admin rpm repo sync run --repo-id srpm-unsigned &
+
+# these are the changes to the rpm_base_4repos shapshot
+pulp-admin rpm repo remove rpm --repo-id rpm-with-modules --str-eq name=bear
+pulp-admin orphan remove --all
+pulp-admin rpm repo update --repo-id rpm-with-modules --feed https://fixtures.pulpproject.org/rpm-unsigned/
+pulp-admin rpm repo update --repo-id rpm-distribution-tree --checksum-type sha1
+pulp-admin rpm repo update --repo-id srpm-unsigned --relative-url srpm
+pulp-admin rpm repo create --feed https://fixtures.pulpproject.org/rpm-richnweak-deps/ --repo-id rpm-richnweak-deps --download-policy on_demand
+# need to wait for tasks to finish
+pulp-admin rpm repo sync run --repo-id rpm-richnweak-deps
+pulp-admin rpm repo remove rpm --repo-id rpm-richnweak-deps --str-eq name=icecubes
```

Comment:
@goosemania not sure if this suggestions is not overkill and maybe out of the scope - how about copy a package from some repo to empty repo where we'd test that repo version is created but unit has not been re-migrated

Reply:
It's a good idea, the copy part.
Let me think if I can copy it somewhere else and not to the empty repo.
I need at least one repo with no changes at all. Thanks for the suggestion!

Reply:
just create 2 empty repos in the first snapshot? in the second snapshot you can still have one empty repo untouched, wdyt?

Reply:
Sure, I was just thinking where to add, without redoing all snapshots.